tests/tx/script/bitcoind.test.ts

```
@@ -0,0 +1,258 @@
+    if (!isNaN(Number(token)) && !token.startsWith('0x')) {
+      const value = BigInt(token);
+      if (value === i64Min) builder.addI64Min();
+      else builder.addI64(value);
+    } else if (token.startsWith('0x')) {
+      const value = Uint8Array.from(Buffer.from(token.slice(2), 'hex'));
+      builder.extend(value);
+    } else if (token.startsWith("'") && token.endsWith("'")) {
+      builder.addData(new TextEncoder().encode(token.slice(1, -1)));
+    } else {
+      const opCodeUpper = token.replace('_', '').toUpperCase();
+      if (opCodeMap.hasOwnProperty(opCodeUpper)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)